web/src/portico/team.ts

```diff
@@ -17,21 +19,25 @@ const repo_name_to_tab_name = {
 
 // Remember the loaded repositories so that HTML is not redundantly edited
 // if a user leaves and then revisits the same tab.
-const loaded_repos = [];
+const loaded_repos: string[] = [];
 
-function calculate_total_commits(contributor) {
+function calculate_total_commits(contributor: Contributor): number {
     let commits = 0;
     for (const repo_name of Object.keys(repo_name_to_tab_name)) {
         commits += contributor[repo_name] || 0;
     }
     return commits;
 }
 
-function get_profile_url(contributor, tab_name) {
+function get_profile_url(contributor: Contributor, tab_name = 0): string | undefined {
```

Comment:
Might need to elaborate on the default value change here. Or maybe we want to just remove `tab_name` in a separate prep commit because [it has been unused since it was added](https://github.com/zulip/zulip/commit/3ec64b60929640eb68001eb3c9b1e50cb8af95a9)?

Reply:
Yes, I think a separate commit removing the parameter `tab_name` from the function would be a good idea. Since, all the function calls don't provide this param, so it should be safe to just remove this param.